ui/react-components/utils/DateUtil.spec.js

```diff
@@ -4,7 +4,7 @@ import{
 
 describe('DateUtil', () =>{
     it('should return true when a valid date is passed', ()=>{
-        const isValid = isValidUSDate('02/12/2019');
+        const isValid = isValidUSDate('12/Feb/2019');
```

Comment:
The method name isValidUSDate( ) is not correct now.. since US date format is usually: mm-dd-yyyy
https://stackoverflow.com/questions/34504131/converting-a-string-date-into-us-format-in-java

I think we should rename this method also.